src/RiskModule.sol

```diff
@@ -78,4 +78,36 @@ library RiskModule {
             liquidationThreshold = uint16(liquidationThreshold256 / totalValue);
         }
     }
+
+    /**
+     * @notice Calculates the collateral value and weighted liquidation threshold given the assets
+     * @param valuesAndRiskVarPerAsset A list of structs with the values and risk variables of each asset address:
+     * - valueInBaseCurrency: The value of the asset denominated in baseCurrency with 18 decimals precision
+     * - collFactor: The Collateral factor or haircut of the asset, to discount the MTM value, 2 decimals precision
+     * - liqThreshold: The Liquidation Threshhold of the asset, 2 decimals precision
+     * @return collateralValue The collateral value of the given assets
+     * @return liquidationThreshold The weighted liquidation threshold of the given assets
+     */
+    function calculateCollateralValueAndLiquidationThreshold(
+        AssetValueAndRiskVariables[] memory valuesAndRiskVarPerAsset
+    ) public pure returns (uint256 collateralValue, uint256 liquidationThreshold) {
+        uint256 totalValue;
+        for (uint256 i; i < valuesAndRiskVarPerAsset.length;) {
+            totalValue += valuesAndRiskVarPerAsset[i].valueInBaseCurrency;
+
+            collateralValue += valuesAndRiskVarPerAsset[i].valueInBaseCurrency * valuesAndRiskVarPerAsset[i].collFactor;
+
+            liquidationThreshold +=
+                valuesAndRiskVarPerAsset[i].valueInBaseCurrency * valuesAndRiskVarPerAsset[i].liqThreshold;
+            unchecked {
+                i++;
```

Comment:
```suggestion
                ++i;
```

for consistency sake and (5 gas or smth), a few other spots where we still have i++